lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2236,6 +2237,249 @@ def test_rescore_problem_all(self, act):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(act.called)
 
+    @patch.dict(settings.FEATURES, {'ENTRANCE_EXAMS': True})
+    def test_course_has_entrance_exam_in_student_attempts_reset(self):
+        """ Test course has entrance exam id set while resetting attempts"""
+        url = reverse('reset_student_attempts_for_entrance_exam',
```

Comment:
Might be better to drop this value to the next line and indent to the first tab stop under 'url = '